src/views/CredentialsMenu/index.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import AccountCircleIcon from '@icons/material/AccountCircleIcon';
+import LoginIcon from '@icons/material/LoginIcon';
+import LogoutIcon from '@icons/material/LogoutIcon';
+
+import AuthContext from '../../components/auth/AuthContext';
+import config from '../../config';
+
+export default class CredentialsMenu extends React.PureComponent {
+  static contextType = AuthContext;
+
+  static handleLoginRequest() {
+    const loginView = new URL(config.redirectRoute, window.location);
+    window.open(loginView, '_blank');
+  }
+
+  componentDidMount() {
+    const { context } = this;
+    if (context) {
+      context.on(
+        'user-session-changed',
+        this.handleUserSessionChanged,
+      );
+    }
+  }
+
+  componentWillUnmount() {
+    const { context } = this.context;
+    if (context) {
+      context.off(
+        'user-session-changed',
+        this.handleUserSessionChanged,
+      );
+    }
+  }
+
+  handleUserSessionChanged = () => {
+    this.forceUpdate();
+  };
+
+  renderWithoutUser = () => <LoginIcon onClick={CredentialsMenu.handleLoginRequest} style={{ cursor: 'pointer' }} />;
+
+  renderWithUser(userSession) {
+    const { context } = this;
+    const icon = userSession.picture ? (
+      <img
+        alt={userSession.name}
+        src={userSession.picture}
+        style={{ width: '2rem', borderRadius: '50%' }}
+      />
+    ) : (
+      <AccountCircleIcon />
+    );
+
+    // XXX: I'm not going with Material UI theming; please advise recommended approach
+    return (
+      <div style={{ display: 'flex', alignItems: 'center' }}>
+        {icon}
+        {userSession.name}
```

Comment:
<img width="157" alt="Screen Shot 2019-04-03 at 3 57 10 PM" src="https://user-images.githubusercontent.com/3766511/55509096-59197800-5629-11e9-900d-9b16a7bec0f4.png">

No need to show the user full name beside the avatar. Also, you can have a menu open with both the user full name and a button to sign out when a user clicks on the avatar.

Reply:
For now I've removed the user. Let me know if this is sufficient or if to file a follow up issue.

Reply:
<img width="115" alt="Screen Shot 2019-04-04 at 1 19 32 PM" src="https://user-images.githubusercontent.com/3766511/55575183-4e6beb00-56dc-11e9-992e-7e5fef9558d1.png">

A couple of things:

1. This logo should be for signing out only, not for signing in. You can use a normal button for the login button.
2. The avatar when logged in gives the impression that it's a button when its sitting beside the logout button.
3. Not everyone knows that a door with an arrow means logout. To improve the UX you can make use of material-ui `Tooltip` component.

> Let me know if this is sufficient or if to file a follow up issue.

I think it makes sense to address it here unless you think otherwise.

Reply:
Since you don't have strong objection I will polish it afterward among few other polishes I have.

Filed https://github.com/mozilla/bugzilla-dashboard/issues/52